README.md

```diff
@@ -412,12 +441,22 @@ axios at the same time.
 }
 ```
 
-All the other parameters will be passed directly to the [`axios.create`](https://github.com/mzabriskie/axios#creating-an-instance) method.
+### RedisStore(client, [, hashKey])
+
+RedisStore allow you to cache requests on server using [redis](https://redis.io/).
+Create a `RedisStore` instance. Takes `client` (`RedisClient`) and optional `hashKey` (name of hashSet to be used in redis).
+
+#### client
+
+```js
+    // Using redis client https://github.com/NodeRedis/node_redis
+    const redis = require("redis")
+    const client = redis.createClient()
+```
 
 #### Returns
 
-`setup()` returns an instance of `axios` pre-configured with the cache adapter.
```

Comment:
Same here, don't remove this part of the readme

Reply:
Done